Octokit.Reactive/Clients/IObservableRepositoryCommentsClient.cs

```diff
@@ -89,5 +89,16 @@ public interface IObservableRepositoryCommentsClient
         /// <param name="number">The comment id</param>
         /// <returns></returns>
         IObservable<Unit> Delete(string owner, string name, int number);
+
+        /// <summary>
+        /// Creates a reaction for specified Commit Comment
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#create-reaction-for-a-commit-comment</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="number">The comment id</param>
+        /// <param name="reaction">The reaction for </param>
```

Comment:
Wording "The reaction for " is a bit odd (and also has an extra space on the end of the string).
Perhaps "The reaction" or "The reaction to create" would be better